lib/routes.js

```diff
@@ -148,10 +132,26 @@ function addControllerRoutes(router) {
 
     controller(pathRouter);
 
-    router.use('/' + routeName, pathRouter);
+    router.use(`/${routeName}`, pathRouter);
   });
 }
 
+/**
+ * Add routes from the authentication module that
+ * can be passed in. If no module is passed in at
+ * instantiation time then use the default auth module.
+ *
+ * @param {express.Router} router
+ */
+function addAuthenticationRoutes(router) {
+  const authRouter = express.Router();
+
+  authRouter.use(require('body-parser').json({ strict: true, type: 'application/json', limit: '50mb' }));
```

Comment:
I've seen a couple of different numbers for the body size limit. I'm not sure how to figure out what it should be, but we should check this number.

Reply:
Yeah, I'm not sure either and I feel `50mb` is a lot, but I think the default value (`100kb`) should be enough or at most `1mb` for the body size limit. What do you think?

Reply:
Here, we have 1mb for nginx and 5mb for Express: https://github.com/clay/clay-starter/blob/master/app/services/startup/index.js but let's check with @jonwinton on this.